EIPS/eip-1559.md

```diff
@@ -114,7 +114,7 @@ class World(ABC):
 		# check if the base fee is correct
 		if parent_gas_used >= parent_gas_target:
 			gas_delta = parent_gas_used - parent_gas_target
-			fee_delta = parent_base_fee * gas_delta // parent_gas_target // BASE_FEE_MAX_CHANGE_DENOMINATOR
+			fee_delta = max(parent_base_fee * gas_delta // parent_gas_target // BASE_FEE_MAX_CHANGE_DENOMINATOR)
```

Comment:
Think we're missing a `, 1` in those max parentheses?

Reply:
Doh!  Totally.  My docker is broken at the moment which is how I test Python stuff... so I just winged it thinking, "I can't possibly get this one line change wrong..."

Reply:
Haha oh yes, I can relate :) it's often the small things which demand the least amount of attention that get messed up!